scenarios/fuel_test/contracts/interaction-tools/src/main.rs

```diff
@@ -17,25 +17,29 @@ async fn main() -> Result<()> {
     // Load .env file
     dotenv().ok();
     let phrase = env::var("MNEMONIC").expect("MNEMONIC must be set in .env");
-    let provider = Provider::connect("testnet.fuel.network").await.unwrap();
+    let provider_url = env::var("PROVIDER_URL").expect("PROVIDER must be set in .env");
+    let base_asset_address = env::var("BASE_ASSET_ADDRESS").expect("BASE_ASSET_ADDRESS must be set in .env");
+    
+    let provider = Provider::connect(provider_url).await.unwrap();
+
     let wallet = WalletUnlocked::new_from_mnemonic_phrase(&phrase, Some(provider.clone())).unwrap();
     let base_asset_id =
-        AssetId::from_str("0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07")
+        AssetId::from_str(&base_asset_address)
             .unwrap();
     let balance = provider
         .get_asset_balance(wallet.address(), base_asset_id)
         .await?;
     if balance < 500000 {
-        println!("Wallet address {}, with balance {}, can use the faucet here: https://faucet-testnet.fuel.network/?address={}", wallet.address(), balance, wallet.address());
+        println!("Wallet address {}, with balance {}, can use the faucet here for testnet: https://faucet-testnet.fuel.network/?address={}", wallet.address(), balance, wallet.address());
         return Ok(());
     }
 
     // Random Salt to deploy a new contract on every run
     let mut salt = [0u8; 32];
     rand::thread_rng().fill(&mut salt);
 
-    let greeter_contract_id =
-        ContractId::from_str("0xb9bc445e5696c966dcf7e5d1237bd03c04e3ba6929bdaedfeebc7aae784c3a0b")?;
```

Comment:
This is commented out, same as below as it's clearly left from running the script at some point.

Reply:
Should we just remove it if it's not useful?

Reply:
removed, it's in the git history if @DZakh you still need it